packer/ansible/rackhd_source.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Development Server
+  hosts: local
+  sudo: no
+  roles:
+    - apt
+    - build
+    - git
+    - rabbitmq
+    - mongodb
+    - node
+    - isc-dhcp-server
+    - monorail
```

Comment:
I don't know what all of this actually is. I do know that rackhd_local.yml works and is the model that I used for the source installer.